docs/reference/evaluation-context/evaluation-context.mdx

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 1
+id: evaluation-context
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Evaluation Context
+
+The _evaluation context_ is a container for arbitrary contextual data that can be used as a basis for dynamic evaluation. Static data such as the host or an identifier for the application can be configured globally. Dynamic evaluation context, such as the IP address of the client in a web application, can be implicitly propagated or explicitly passed to during flag evaluation, and can be merged with static values. Many flag management systems support the definitions of targeting rules which allow for
```

Comment:
This paragraph ends prematurely.

Reply:
Sorry, I forgot to push a commit. I moved this to a later section.